networkx/algorithms/min_edge_cover.py

```diff
@@ -0,0 +1,63 @@
+#TODO: Add Test Cases
+"""
+An edge cover of a graph is a set of edges such that every vertex of
+the graph is incident to at least one edge of the set.
+A minimum edge cover is an edge covering of smallest possible size.
+
+https://en.wikipedia.org/wiki/Edge_cover
+
+"""
+import networkx as nx
+__all__ = ["min_edge_cover"]
+__author__ = """Nishant Nikhil (nishantiam@gmail.com)"""
```

Comment:
What has been deprecated and how should @nishnik replace this with the new way to do it?